oz-to-ml-calculator.js

```diff
@@ -0,0 +1,4 @@
+const ozToMl = (oz) => {
+  return `${oz * 29.5735}`;
```

Comment:
Here you are unnecessarily converting a number to a string. You could just return the number instead:

```suggestion
  return oz * 29.5735;
```